boards/shields/st_b_lcd40_dsi1_mb1166/boards/stm32h747i_disco_stm32h747xx_m7.overlay

```diff
@@ -61,15 +61,3 @@
 	hs-active-high;
 	de-active-high;
 };
-
-&i2c4 {
-	pinctrl-0 = <&i2c4_scl_pd12 &i2c4_sda_pd13>;
-	pinctrl-names = "default";
-	clock-frequency = <I2C_BITRATE_FAST>;
-	status = "okay";
-
-	ft5336: ft5336@38 {
-		compatible = "focaltech,ft5336";
-		reg = <0x38>;
-	};
-};
```

Comment:
This should be kept here.

Reply:
ft5336 touch controller is available in the panel, so added in the panel shield.

Reply:
Yes, but the i2c device, that will be used with it, is board dependent.